### PR TITLE
[DO NOT MERGE] freon 2 electric boogaloo but proper

### PR DESCRIPTION
**Pull request by SireTurret**
## Changelog
:cl: Turret
add: Removed Hyper Nobilium and instead brought back Freon, the old gas which was better.
/:cl:

## About The Pull Request

Hypernobilum was removed from the game, instead it's been replaced by Freon. That's it.

## Why It's Good For The Game

Removing Freon was an unnecesary, /tg/ change.

### CODE REVIEW

**Comment by ghost** [comment]
merge it [s] now

**Comment by SireTurret** [comment]
dont

**Comment by SireTurret** [comment]
im gonna actually die

File: hippiestation/code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 1]
```diff
@@ -0,0 +1,167 @@
+GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide, /datum/gas/plasma)) //the main four gases, which were at one time hardcoded
```
Comment (yoyobatty) [inline hippiestation/code/modules/atmospherics/gasmixtures/gas_types.dm:1]: You just copy pasted the original gas_types.dm file.

I already have a working freon branch, if you want, I can PR that and you can add to that.

**Review by yoyobatty — CHANGES_REQUESTED** [review]
Fix this or I will PR my working branch

**Comment by SireTurret** [comment]
I think you PR'ing your working branch would be better, since I'm really a monkey with a grenade on this.

**Pull request closed without merging**